src/handlers/miner_hbbft_handler.erl

```diff
@@ -15,6 +15,7 @@
         ]).
 
 -include_lib("blockchain/include/blockchain_vars.hrl").
+-include_lib("helium_proto/include/blockchain_block_pb.hrl").
```

Comment:
Is there a way to add accessors to blockchain_block or whatever to avoid this?

Reply:
Yes. Will do.